app/assets/javascripts/components/revisions/revision.jsx

```diff
@@ -4,14 +4,20 @@ import DiffViewer from './diff_viewer.jsx';
 import CourseUtils from '../../utils/course_utils.js';
 import { formatDateWithTime } from '../../utils/date_utils.js';
 import { trunc } from '../../utils/strings.js';
+import withRouter from '@components/util/withRouter.jsx';
 
-const Revision = ({ revision, index, wikidataLabel, course, setSelectedIndex, lastIndex, selectedIndex, student }) => {
+const Revision = ({ revision, index, wikidataLabel, course, setSelectedIndex, lastIndex, selectedIndex, student, router }) => {
   const ratingClass = `rating ${revision.rating}`;
   const ratingMobileClass = `${ratingClass} tablet-only`;
   const formattedTitle = CourseUtils.formattedArticleTitle({ title: revision.title, project: revision.wiki.project, language: revision.wiki.language }, course.home_wiki, wikidataLabel);
   const subtitle = wikidataLabel ? `(${CourseUtils.removeNamespace(revision.title)})` : '';
   const isWikipedia = revision.wiki.project === 'wikipedia';
   const showRealName = student.real_name;
+
+  const openStudentDetails = () => {
+    const url = `/courses/${course.slug}/students/articles/${encodeURIComponent(student.username)}`;
```

Comment:
this same logic is used [here](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/14747b1b99b137031d7c54f8636360a1d9dab1d5/app/assets/javascripts/components/students/shared/StudentList/Student/Student.jsx#L49), perhaps make a utility function to abstract it away?

Reply:
That's a good idea. I'll file a separate issue for it after this is merged.